src/constraint_system/ecc/scalar_mul/fixed_base.rs

```diff
@@ -28,20 +28,21 @@ where
         multiples[i] = multiples[i - 1].double();
     }
 
-    ProjectiveCurve::batch_normalization_into_affine(&mut multiples)
+    ProjectiveCurve::batch_normalization_into_affine(&multiples)
 }
 
 impl<E: PairingEngine, P: TEModelParameters<BaseField = E::Fr>>
     StandardComposer<E, P>
 {
     /// Adds an elliptic curve Scalar multiplication gate to the circuit
     /// description.
-    ///
-    /// # Note
-    /// This function is optimized for fixed base ops **ONLY** and therefore,
-    /// the **ONLY** `generator` inputs that should be passed to this
-    /// function as inputs are [`dusk_jubjub::GENERATOR`] or
-    /// [`dusk_jubjub::GENERATOR_NUMS`].
+    // FIXME: `dusk_jubjub` is not used in this crate.
+    //
+    // # Note
+    // This function is optimized for fixed base ops **ONLY** and therefore,
+    // the **ONLY** `generator` inputs that should be passed to this
+    // function as inputs are [`dusk_jubjub::GENERATOR`] or
+    // [`dusk_jubjub::GENERATOR_NUMS`].
```

Comment:
Edit the coment and fix it directly instead of leaving the FIXME please. :)

Reply:
I wrote this because I didn't know what the new comment should be.

Reply:
Something like:
```
    // # Note
    // This function is optimized for fixed base ops **ONLY** and therefore,
    // the **ONLY** input that should be passed to the function as a point is the
    // generator or basepoint of the curve over which we are operating.
```
should do. :smile: